support/addons.js

```diff
@@ -2,7 +2,9 @@
 
 const chai = require('chai');
 const chaiAsPromised = require('chai-as-promised');
+const chaiUuid = require('chai-uuid');
```

Comment:
is this being used anywhere?

Reply:
No, it is not anymore. Sloppy sloppy.

I'll remove.